public/locales/cn/auth.json

```diff
@@ -128,5 +128,34 @@
   },
   "logout": {
     "message": "下次再见啦~"
+  },
+  "contact-form": {
+    "title": "您的意见对我们非常宝贵, 欢迎联系我们 👋",
+    "customername": {
```

Comment:
maybe use `customer-name`, just follow the naming convention in the other parts of this JSON file